### PR TITLE
Message Wall Pin

### PR DESCRIPTION
**Pull request by rossdotparker**
**Description**
Allows users with Manage Messages_all to pin messages to the top of the message wall. These messages are highlighted in purple.

**Motivation and Context**
All messages are equal, but some are more equal than others.

**How Has This Been Tested?**
Locally

**Screenshots**
Screenshot 2019-11-12 at 9 53 24 AM

### CODE REVIEW

File: modules/Messenger/messenger_manage_edit.php [hunk 1]
```diff
@@ -147,7 +153,9 @@
 					$smsURL=getSettingByScope( $connection2, "Messenger", "smsURL" ) ;
 					$smsURLCredit=getSettingByScope( $connection2, "Messenger", "smsURLCredit" ) ;
 					if ($smsUsername == "" OR $smsPassword == "" OR $smsURL == "") {
-						$form->addRow()->addAlert(sprintf(__('SMS NOT CONFIGURED. Please contact %1$s for help.'), "<a href='mailto:" . $_SESSION[$guid]["organisationAdministratorEmail"] . "'>" . $_SESSION[$guid]["organisationAdministratorName"] . "</a>"), 'error');
+						$row = $form->addRow()->addClass('sms');
+							$row->addLabel('sms', __('SMS'))->description(__('Deliver this message to user\'s mobile phone?'));
```
Comment (SKuipers) [inline modules/Messenger/messenger_manage_edit.php]: Is this label meant to be on the non-configured side of this conditional, or the configured side? I suspect maybe the latter 🤔

Comment (rossdotparker) [reply modules/Messenger/messenger_manage_edit.php]: Yes, you are right. Are you OK to tweak? Thanks!

Comment (SKuipers) [reply modules/Messenger/messenger_manage_edit.php]: 👍

**Comment by SKuipers** [comment]
> All messages are equal, but some are more equal than others.
😂

**Pull request merged**